tests/test_service.py

```diff
@@ -177,7 +187,7 @@ def test_invalid_operation(self):
         """ Testing invalid HTTP operation """
         created_wishlist = Wishlist(customer_id=1, name="wishlist")
```

Comment:
if you're changing it to post, we don't need to have the two lines with `created_wishlist`

Reply:
Yes, I have a valid operation before an invalid one, so it stays.

Reply:
why do we need the valid operation when its response is not being validated anyway?

Reply:
to have a valid operation before the invalid one.

Reply:
I'm sorry. I think I'm missing something. Why do we need a valid operation before the invalid one?

Reply:
It became valid when I added the get. So I only added the invalid one, to make it pass the test.

Reply:
Okay. Understood. So we don't need to perform the valid get operation in the test for invalid. We can remove the following three lines, right? Or will something fail?

https://github.com/nyu-devops-fall19-wishlists/wishlists/pull/190/files#diff-c95766f217f901a70ccf06e3586e8218R198-R200

Reply:
Please go ahead and change it as you wish.